File: app/src/main/java/gardezi/io/amiibofinder/dagger/FragmentBuildersModule.java
package gardezi.io.amiibofinder.dagger;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;
import gardezi.io.amiibofinder.view.AmiiboListFragment;

@Module
public abstract class FragmentBuildersModule {
    @ContributesAndroidInjector
    abstract AmiiboListFragment contributeAmiiboListFragment();

}

File: app/src/main/java/gardezi/io/amiibofinder/dagger/FragmentBuildersModule.kt
package gardezi.io.amiibofinder.dagger

import dagger.Module
import dagger.android.ContributesAndroidInjector
import gardezi.io.amiibofinder.view.AmiiboListFragment

@Module
abstract class FragmentBuildersModule {
    @ContributesAndroidInjector
    internal abstract fun contributeAmiiboListFragment(): AmiiboListFragment

}
